Lib/idlelib/configdialog.py

```diff
@@ -683,6 +689,7 @@ def var_changed_space_num(self, *params):
         "Store change to indentation size."
         value = self.space_num.get()
         changes.add_option('main', 'Indent', 'num-spaces', value)
+        self.after_idle(self.indent_chooser.selection_clear)
```

Comment:
Is this about the box being readonly?  It does not clear the currently selected value.